src/routes/Tool/Tool.tsx

```diff
@@ -290,20 +284,27 @@ const Tool = () => {
         }
       />
 
+      <div className="mb-8 mt-5">
+        <Button
+          onClick={() => void updateEhdotukset()}
+          label={t('tool.update-job-opportunities-list')}
+          variant="accent"
+          disabled={ehdotuksetLoading}
+        />
+      </div>
+
       <div className="grid grid-cols-1 gap-x-6 sm:grid-cols-3">
         <div className="col-span-1 sm:col-span-3">
-          <div className="mt-10 flex flex-col sm:mt-11 sm:flex-row">
-            <span className="text-body-md font-arial text-black font-medium sm:text-body-lg sm:font-poppins">
-              {t('tool.competences.available-options')}{' '}
-              <span className="text-heading-3 font-bold">
-                {t('tool.competences.available-options-totals', { professionsCount, educationsCount })}
-              </span>
+          <div className="flex flex-col sm:flex-row">
+            <span className="text-body-md text-black font-medium sm:text-body-lg">
+              <h2 className="text-heading-2 mb-4">{t('tool.competences.opportunities-title')}</h2>
+              {t('tool.competences.available-options-totals', { professionsCount, educationsCount })}
             </span>
           </div>
         </div>
 
-        <div className="col-span-1 mt-10 flex flex-col sm:col-span-3 sm:mt-9 sm:flex-row">
-          <span className="text-body-md font-arial text-black font-medium sm:text-body-lg sm:font-poppins">
+        <div className="col-span-1 mt-5 flex flex-col sm:col-span-3 sm:flex-row">
+          <span className="text-body-md text-black font-medium sm:text-body-lg">
             {t('tool.competences.adjust-data-emphasis')}
```

Comment:
As these slider's values are no longer checked in useEffect; changing Sliders alone won't do anything. It requires to press the new update button.

Should the text be updated to reflect it?
Should the update button to actually be below sliders?
Should updating the values of sliders still make a update to opportunities list, separately from button press?

Reply:
As discussed separately these pressing matters can be brought to next PAMU discussion.